### PR TITLE
Update Json.NET to 9.0.1-beta1

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by cesarbs** [comment]
🚢

**Comment by halter73** [comment]
:shipit:

File: test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
     "Microsoft.AspNetCore.Server.Kestrel.Https": "1.0.0-*",
     "Microsoft.AspNetCore.Testing": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "9.0.1-beta1",
```
Comment (jasonwilliams200OK) [inline test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json]: While you are at it, please remove `"portable-dnxcore50+net45+win8+wp8+wpa81"` from `"imports"` array. That is not needed anymore.

Comment (javiercn) [inline test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json]: @jasonwilliams200OK We are doing another pass after it's updated to do that.

Comment (jasonwilliams200OK) [inline test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json]: Great! Once xunit.assert gets ported to net standard, then we can just get rid off the entire import chain! :smile:

Comment (javiercn) [inline test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json]: @jasonwilliams200OK That's the dream :smile:

**Pull request merged**